report/sections/Chapter 01 - Introduction/Intro.tex

```diff
@@ -38,7 +37,7 @@ \section{Existing solutions}
 The feedback loop for each platform is quite different, however.
 
 
-Codewars focuses on competitive programming, meaning users compete to solve problems fastest and most efficient way.
+Codewars focuses on competitive programming, meaning users compete to solve problems the fastest and most efficient way.
```

Comment:
Er det fastest and most efficient der er bedst? Mig bekendt er det andre users der rater ens losning.